cmd/users/main.go

```diff
@@ -313,31 +314,79 @@ func newService(db *sqlx.DB, tracer opentracing.Tracer, auth mainflux.AuthServic
 			Help:      "Total duration of requests in microseconds.",
 		}, []string{"method"}),
 	)
-	if err := createAdmin(svc, userRepo, c); err != nil {
+	if err := createAdmin(svc, userRepo, auth, c); err != nil {
 		logger.Error("failed to create admin user: " + err.Error())
 		os.Exit(1)
 	}
 	return svc
 }
 
-func createAdmin(svc users.Service, userRepo users.UserRepository, c config) error {
+func createAdmin(svc users.Service, userRepo users.UserRepository, auth mainflux.AuthServiceClient, c config) error {
+	authoritiesObjKey := "authorities"
+	memberRelationKey := "member"
+
 	user := users.User{
 		Email:    c.adminEmail,
 		Password: c.adminPassword,
 	}
 
-	if _, err := userRepo.RetrieveByEmail(context.Background(), user.Email); err == nil {
-		// Exiting if user already exists
+	if u, err := userRepo.RetrieveByEmail(context.Background(), user.Email); err == nil {
+		// If the Admin is existing, check whether it has proper policies or not.
```

Comment:
If admin is created but policies failed maybe we can recreate admin on next startup
check this https://github.com/mainflux/mainflux/blob/0fe9f55bae8d0887d5ea2d7be9fb483c6b0bfafc/provision/service.go#L103
`recover` function does cleanup in case of error